x/subscription/spec/02_state.md

```diff
@@ -0,0 +1,35 @@
+<!--
+order: 2
+-->
+
+# State
+
+The `x/subscription` stores the plans and subscriptions on chain:
+
+```golang
+type Plan struct {
+  id int,  // auto-increasing unique identifier
+  title string,
+  description string,
+  owner Address,  // beneficial owner of the plan
+  price Decimal,  // price in fiat currency show to user
+  fiat_currency String,  // fiat currency name
+  token_denom String,  // denomation of token
+  conversion_rate Decimal,  // conversion rate from fiat currency to token, maintained by owner
+
+  interval int, // number of seconds of payment period
+  timezone Timezone, // the timezone used to offset the time interval
```

Comment:
timezones are definitely needed on the client / UX side -- for this network-only state, Unix time is enough (one can just use the interval and display it with the respect to the local settings) or will it create any problems?

Reply:
It's for calculating a period index for local timezone, since `block_time / interval` is different from `(block_time + 8hours) / interval`.

Reply:
changed `timezone` to more general `interval_offset`, to specify intervals more flexible.